libs/blocks/global-navigation/global-navigation.css

```diff
@@ -650,6 +654,10 @@ header.global-navigation {
   .feds-brand-image + .feds-brand-label {
     display: none;
   }
+
+  .feds-menu-container {
+    width: 100%;
+  }
```

Comment:
Does this need to be so specific between the 2 breakpoints? Could we not set `min-width: 100%;` starting from mobile devices, which would then be overwritten on desktop?